dsl_parser/functions.py

```diff
@@ -156,8 +168,7 @@ def parse_args(self, args):
             raise ValueError(
                 "Illegal argument(s) passed to {0} function. Expected either"
                 "a string or a list [input_name [, key1/index1 [,...]]] but "
-                "got {1}".format(self.name, ','.join(
-                    [str(arg) for arg in args])))
+                "got {1}".format(self.name, args))
```

Comment:
Why is the str cast is not necessary anymore?

Reply:
I've tested in `ipython` and the output is basically the same.
```
In [1]: a = [u'one',2,3]

In [2]: str(a)
Out[2]: "[u'one', 2, 3]"

In [3]: ','.join([str(i) for i in a])
Out[3]: 'one,2,3'
```